cinema/models.py

```diff
@@ -5,6 +5,32 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    genres = models.ManyToManyField("Genre", blank=True, related_name="genres")
```

Comment:
The `related_name` for the `genres` field in the `Movie` model should be unique and descriptive. Using 'genres' as the related name can cause confusion and potential conflicts. Consider using something like 'movies' to indicate the reverse relationship from `Genre` to `Movie`.